tests/src/test/java/testconfig/PrintOutCurrentTestRunListener.java

```diff
@@ -0,0 +1,32 @@
+package testconfig;
+
+import org.junit.runner.Description;
+import org.junit.runner.Result;
+import org.junit.runner.notification.RunListener;
+
+public class PrintOutCurrentTestRunListener extends RunListener {
+    @Override
+    public void testRunStarted(Description description) {
+        System.out.println("testRunStarted " + description.getClassName() + " " + description.getDisplayName() + " "
+                + description.toString());
+    }
+
+    public void testStarted(Description description) {
+        System.out.println("testStarted "
+                + description.toString());
+    }
+
+    public void testFinished(Description description) {
+        System.out.println("testFinished "
+                + description.toString());
+    }
+
+    public void testRunFinished(Result result) {
+        System.out.println("testRunFinished " + result.toString()
+                + " time:"+result.getRunTime()
+                +" R"+result.getRunCount()
+                +" F"+result.getFailureCount()
+                +" I"+result.getIgnoreCount()
```

Comment:
Can we add a colon similar to time in line 26 for better readability? And maybe a space after that?

Reply:
done https://github.com/factset/analyticsapi-engines-java-sdk/pull/63/commits/453ecaead622869608e9edf4587f442e97649dbd